### DISCUSSION TITLE
custom runner as service failed to start

### DISCUSSION THREAD

**Discussion by hannesa2** [community/community#48686] [Actions]
### Select Topic Area

Question

### Body

I have an arm64 Mac which host several custom runners for different repos.

It was working nice, but month ago, the runner was removed in repository and I was forced to add it again. But till then I can't run it as service any more
[installing as service](https://docs.github.com/en/actions/hosting-your-own-runners/configuring-the-self-hosted-runner-application-as-a-service#installing-the-service) runs into an issue.

<img width="1307" alt="image" src="https://user-images.githubusercontent.com/3314607/221514950-4717d37c-9f37-4d94-920d-4d23ad6ea269.png">

Any idea how to solve this ?

**Comment by ADTC** [comment]
Quoting here, for anyone who may need it: https://github.com/actions/runner/issues/1056#issuecomment-1201904324

> It would be great if there were a purely ssh solution that didn't depend on the GUI at all. In the meantime, this can be helpful: go to System Preferences -> Users and Groups -> Login Options -> Automatic Login, and set the 'runner' or 'gha' user to login automatically. Reboot. The user will login at boot time and the services will start.
